src/image-sync-manager.ts

```diff
@@ -1,7 +1,26 @@
+// @ts-ignore
+import {cuepoint} from 'kaltura-player-js';
 import {ImagePlayer} from './image-player';
 
-export const TIMED_METADATA_ADDED = 'timedmetadataadded'; // TODO: should be enum from cuePointManager
-export const SLIDE_CUE_POINT_TYPE = 'slides'; // TODO: 'slides' should be enum from cue-point plugin
+const ThumbCuePointType = 'thumbCuePoint.Thumb'; // TODO: use enum from cue-point service once it got deployed
+
+interface TimedMetadata {
+  payload: {
+    cues: Array<{
+      track: {
+        label: string;
+      };
+      value: {
+        data: {
+          id: string;
+          cuePointType: string;
+          assetUrl: string;
+        };
+        key: string;
+      };
+    }>;
+  };
+}
```

Comment:
shouldn't this be in kaltura-cuepoints plugin?

Reply:
yep, that's why we have TODO here :)

Reply:
sounds good, I mean both enum and interface of course